src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -135,6 +136,16 @@ private void initialize() {
         setUpKeyBindings();
     }
 
+    private String getAllFiles() {
```

Comment:
It DOES NOT return a list of all files.

Mabe following method name is better?

```suggestion
    private String getToolTip() {
```

Reply:
BTW: I think, this is TOTALLY WRONG.

Before:

![image](https://github.com/user-attachments/assets/21d5a683-24ec-4b3e-8709-6388c3e024fc)

After:

![image](https://github.com/user-attachments/assets/20df2d2a-6704-44bb-8637-5dce628530a7)

Hover ON A SINGLE FILE shows ALL FILES.

-- I think ALL THANGES in LinkedFilesEditor have to be reverted!!!